apps/site/pages/en/blog/migrations/v12-to-v14.mdx

```diff
@@ -0,0 +1,45 @@
+---
+date: '2025-09-26T12:00:00.000Z'
+category: migrations
+title: Node.js v12 to v14
+layout: blog-post
+author: AugustinMauroy
+---
+
+# Node.js v12 to v14
+
+<AlertBox level="warning" title="!">
+  This article cover a part of the migration from Node.js v12 to v14. The
+  userland migrations team is working on more codemods to help you with the
+  migration.
+</AlertBox>
```

Comment:
Does this really have to be an alert box?

Reply:
That doesn't need to be a warning? It's not urgent, it can easier be part of the article

Reply:
I think the warning is a bit much 😅 Is there an info box instead of a warning?

Reply:
@JakobJingleheimer info will be blue. what do you think ?

Reply:
Perfect 🙂

Reply:
Why can't it just be a block of text, why show an alert box at all?

Reply:
I think a slight attention-grab could be good, but if it's a choice between warning and plain text, I would say plain text. Between info and plain text, I'm +0 for blue/info.

Reply:
I really think plain text is better. We don't have a single other Markdown article / blog using an alert box, and it feels weird to use it as such.

Reply:
So simple markdown paragraph of quote block ?